src/core/plugins/core-factions/webview/Factions.vue

```diff
@@ -31,9 +31,15 @@ import { defineComponent, defineAsyncComponent } from 'vue';
 
 import { ExampleFactionData } from './utility/exampleFactionData';
 import { Faction } from '../shared/interfaces';
+import { FactionParser } from './utility/factionParser';
 import { FACTION_EVENTS } from '../shared/factionEvents';
 import { Vector3 } from '../../../shared/interfaces/vector';
 
+interface iPage {
```

Comment:
Should be IPage instead of iPage

Reply:
well, that part was CopyPasted from [here](https://github.com/Stuyk/altv-athena/blob/8e140d511cbb8c38d182a8150e51f5d31a13db57/src/core/plugins/core-factions/webview/components/Navigation.vue#L23) so blame @Stuyk